buildGradleApplication/default.nix

```diff
@@ -14,13 +13,15 @@
   version,
   src,
   meta,
+  jdk ? pkgs.jdk,
+  gradle ? pkgs.gradle,
```

Comment:
I guess that's reasonable in some cases. However, in most cases, overlays is probably the way to go.

Reply:
I don't know why I would use overlays here tbh.
That change was able to work for me with 0 issues.

Reply:
sure, it works perfectly fine. Whenever possible, I prefer to have a single version of a JDK for - and by setting it explicitly in the overlay, it's used everywhere and not just for this package. But it's a matter of the use case I suppose 🤷.